carthage.mk

```diff
@@ -0,0 +1,44 @@
+ifneq ($(PROCURSUS),1)
+$(error Use the main Makefile)
+endif
+
+ifneq (,$(findstring darwin,$(MEMO_TARGET)))
+
+SUBPROJECTS      += carthage
+CARTHAGE_VERSION   := 0.37.0
+DEB_CARTHAGE_V     ?= $(CARTHAGE_VERSION)
+
+carthage-setup: setup
+	wget -q -O $(BUILD_SOURCE)/Carthage-$(CARTHAGE_VERSION).tar.gz https://github.com/Carthage/Carthage/archive/$(CARTHAGE_VERSION).tar.gz
```

Comment:
You can use the new as2 branch GITHUB_ARCHIVE Format instead of wget directly for github.

Reply:
This has been resolved in the latest commit.